docs/vm/backup-restore.md

```diff
@@ -116,6 +116,50 @@ EOF
 ### Restore a new VM in a new cluster
 
 1. Setup the same backup target in a new cluster. And the backup controller will automatically sync the backup metadata to the new cluster.
-2. Go to the `Backups` page.
+2. Go to the `VM Backups` page.
 3. Select the synced VM backup metadata and choose to restore a new VM with a specified VM name.
 4. A new VM will be restored using the backup volumes and metadata. You can access it from the `Virtual Machines` page.
+
+# VM Snapshot & Restore
+
+_Available as of v1.1.0_
+
+VM snapshots are created from the **Virtual Machines** page. The VM snapshot volumes will be stored in the cluster, and they can be used to either restore a new VM or replace an existing VM.
+![vm-snapshot.png](/img/v1.1/vm/vm-snapshot.png)
+
+## Create a VM snapshot
+
+1. Go to the `Virtual Machines` page.
+1. Click `Take VM Snapshot` of the VM actions to create a new VM snapshot.
+1. Set a custom snapshot name and click `Create` to create a new VM snapshot.
+![create-snapshot.png](/img/v1.1/vm/create-snapshot.png)
+
+**Result:** The snapshot is created. You can also go to the `Backup & Snapshot > VM Snapshots` page to view all VM snapshots.
+
+The `ReadyToUse` status will be set to `true` once the Snapshot is complete.
+
+Users can either choose to restore a new VM or replace an existing VM using this snapshot.
+
+## Restore a new VM using a snapshot
+
+To restore a new VM from a snapshot, follow these steps:
+
+1. Go to the `VM Snapshots` page.
+1. Specify the new VM name and click `Create`.
+1. A new VM will be restored using the snapshot volumes and metadata, and you can access it from the `Virtual Machines` page.
+![restore-vm-snapshot.png](/img/v1.1/vm/restore-vm-snapshot.png)
+
+## Replace an Existing VM using a snapshot
+
+You can replace an existing VM using the snapshot.
+
+:::note
+
+You can only choose to retain the previous volumes.
+
+:::
+
+1. Go to the `VM Snapshots` page.
+1. Click `Create`.
```

Comment:
didn't find the create option?
<img width="2195" alt="image" src="https://user-images.githubusercontent.com/4569037/197689566-c4b46c9a-82da-478e-be7a-4e4c9d1ebb78.png">

Reply:
Updated it. Thank you.